mindsdb/integrations/handlers/twelve_labs_handler/settings.py

```diff
@@ -90,6 +94,33 @@ def check_param_typos(cls, values):
 
         return values
 
+    @root_validator(pre=True, allow_reuse=True, skip_on_failure=True)
+    def check_for_valid_engine_options(cls, values):
```

Comment:
Can we do the same for task: check for => summarization or search

Reply:
Or this is already handled by the type check of the task

Reply:
This was the handled within the handler, but it makes more sense to move it here, so I added another task for it.